docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,108 @@
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of `GROUP BY`, `DISTINCT`, and other hashing operations. ({issue}`19059`)
+* Fix query failure due to exceeding node memory limits with `INSERT`
+  statements. ({issue}`18771`)
+* Fix incorrectly reporting `ns` instead of `ms` in query plans. ({issue}`19047`)
+
+## Security
+
+* Add access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+* Add support for writer
+  [version 7](https://docs.delta.io/latest/versioning.html#features-by-protocol-version). ({issue}`15873`)
+* Add support for writing the `timestamp` type. ({issue}`16927`)
+* Reduce data read from Parquet files for queries with filters. ({issue}`19032`)
+* Improve performance of writing to Parquet files. ({issue}`19122`)
+* Fix error reading Delta Lake table history when the initial transaction logs
+  have been removed. ({issue}`18845`)
+
+## Elasticsearch connector
+
+* Fix query failure when a `LIKE` clause contains multi-byte characters. ({issue}`18966`)
+
+## Hive connector
+
+* Add support for `double` to `varchar` coercion in Hive tables. ({issue}`18930`)
+* Add automatic type coercion from `varchar(0)` to `varchar(1)`. ({issue}`18811`)
+* Add support for altering column comments in the Glue catalog. ({issue}`19076`)
+* Reduce data read from Parquet files for queries with filters. ({issue}`19032`)
+* Improve performance of reading text files. ({issue}`18959`)
+* Remove legacy Hive readers and writers. The `*_native_reader_enabled` and
+  `*_native_writer_enabled` session properties and `*.native-reader.enabled` and
+  `*.native-writer.enabled` configuration properties are removed. ({issue}`18241`)
+* Remove support for S3 Select. The `s3_select_pushdown_enabled` session
+  property and the `hive.s3select*` configuration properties are removed. ({issue}`18241`)
+* Remove support for disabling optimized symlink listing. The
+  `optimize_symlink_listing` session property and
+  `hive.optimize-symlink-listing` configuration property are removed. ({issue}`18241`)
+* Fix incompatibility with Hive OpenCSV deserialization. As a result, when the
+  escape character is explicitly set to `"`, a `\` (backslash) must be used
+  instead. ({issue}`18918`)
+* Fix performance regression when reading CSV files on AWS S3. ({issue}`18976`)
+
+## Hudi connector
+
+* Fix query failure when reading from Hudi tables with
+  [`instants`](https://hudi.apache.org/docs/concepts/#timeline) that have been
+  replaced. ({issue}`18213`)
+
+## Iceberg connector
+
+* Add support for usage of `date` and `timestamp` arguments in `FOR TIMESTAMP AS
+  OF` expressions. ({issue}`14214`)
+* Add support for using tags with `AS OF VERSION` queries. ({issue}`19111`)
+* Reduce coordinator memory usage when running queries on tables with large
```

Comment:
large metadata .. that sounds weird .. is there some better term? a lot of metadata .. is also weird